docs/dev/dbsync.rst

```diff
@@ -0,0 +1,49 @@
+.. _db-autosync:
+
+Auto Sync of DB tables so we can run MR and AT in parallel
+===========================================================
+
+Scripts run daily on Jenkins to sync 12 tables from modeling-epi-db with epiat-db-p01
+and epi-db-d01.  The 12 tables are:
+
+* epi.modelable_entity
```

Comment:
Every line here, the db table name should be in double-backquotes so it doesn't get interpreted as italic.

Reply:
ok.